### PR TITLE
VSS-100 configure sass

### PR DESCRIPTION
**Pull request by 98arun**
- Configure the sass
- Add the base file in the sass folder
- Following the 7-in-1 structure of sass
- All the files are empty

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
 </head>
 
 <body>
+    <h1>hey</h1>
```
Comment (xDAnkit) [inline index.html]: Hello @98arun , Can we add a wrapper or div container above `h1` tag to wrap the element in a proper parent element

File: index.html [hunk 2]
```diff
@@ -11,6 +11,7 @@
...
     <script src="./assets/js/index.js"></script>
```
Comment (xDAnkit) [inline index.html]: Can we move this `./assets/js/index.js` after the body tag

**Review by xDAnkit — CHANGES_REQUESTED** [review]
Kindly check my comments

File: index.html (hunk 1 above)
Comment (98arun) [reply index.html]: Hello @xDAnkit, that would be a good approach. Let me do that

File: index.html (hunk 2 above)
Comment (98arun) [reply index.html]: Hey @xDAnkit, yes I'll be moving it to the bottom

**Review by xDAnkit — APPROVED** [review]
Looks good

File: index.html (hunk 2 above)
Comment (xDAnkit) [reply index.html]: Thank you @98arun for considering the following changes

**Pull request merged**